Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionDataRecord.h

```diff
@@ -55,6 +76,15 @@ NS_SWIFT_NAME(_WKWebExtension.DataRecord)
 /*! @abstract The total size of all data types contained in this data record. */
 @property (nonatomic, readonly) unsigned long long totalSize;
 
+/*! @abstract The error set if an issue occurred when either deleting or calculating local storage. */
+@property (nonatomic, nullable, readonly) NSError *localStorageError;
+
+/*! @abstract The error set if an issue occurred when either deleting or calculating session storage. */
+@property (nonatomic, nullable, readonly) NSError *sessionStorageError;
+
+/*! @abstract The error set if an issue occurred when either deleting or calculating sync storage. */
+@property (nonatomic, nullable, readonly) NSError *syncStorageError;
```

Comment:
As mentioned, I think `@property (nonatomic, readonly, copy) NSArray<NSError *> *errors` will be better here since you have individual error codes per type already.